assets/app/view/buy_trains.rb

```diff
@@ -9,6 +9,56 @@ module View
   class BuyTrains < Snabberb::Component
     include Actionable
     needs :show_other_players, default: nil, store: true
+    needs :selected_corporation, default: nil, store: true
+
+    def can_buy_others
```

Comment:
this is actually not true for chesapeake

Reply:
Gotcha will look and fix

Reply:
we'd have to change the game logic too, which is why i'm wafflin

Reply:
Yep, it'll need to move down a layer then.